README.md

```diff
@@ -1,25 +1,25 @@
 rOpenSci .github files
 ======================
 
-These files are mostly not specific to rOpenSci - there are a few items in the files specific to rOpenSci, but can be easily removed.
+These templates are part of our guidance for package authors. Cf [our gitbook](https://ropensci.github.io/dev_guide/) in particular the [chapter about collaboration](https://ropensci.github.io/dev_guide/collaboration.html).
 
 Includes:
 
-* **CONTRIBUTING.md**
-* **issue_template.md** (see the [raw text](https://raw.githubusercontent.com/ropensci/dotgithubfiles/master/issue_template.md))
-* **pull_request_template.md** (see the [raw text](https://raw.githubusercontent.com/ropensci/dotgithubfiles/master/pull_request_template.md))
+* [**CONTRIBUTING.md**](dotgithub/CONTRIBUTING.md)
+* [**issue_template.md**](dotgithub/issue_template.md)
+* [**pull_request_template.md**](dotgithub/pull_request_template.md)
 
 Steps:
 
-* Put these three files in a `.github` directory in the base of your R package.
+* Put the [dotgithub folder](dotgithub/) containing these three files in a `.github` directory in the base of your R package.
```

Comment:
we should specify to not put the `dotgithub` folder inside the `.github` folder, BUT instead the files within `dotgithub` within `.github`

Reply:
rodev has a helper function now so one doesn't need to think but good point